README.md

```diff
@@ -127,33 +127,57 @@ The architecture can be further enhanced via:
 
 ## Rust Operator instructions
 
-### Automated deployment (uses existing state file)
+### Anvil Deployment
 
-1. Run `make start-chain-with-contracts-deployed`
-    - This will build the contracts, start an Anvil chain, deploy the contracts to it, and leaves the chain running in the current terminal
+1. Start Anvil Chain
 
-2. Run `make start-rust-operator`
+In terminal window #1, execute the following commands:
+```sh
+anvil
+```
 
-3. Run `make spam-rust-tasks`
+2. Deploy Contracts
 
-Tests are supported in anvil only . Make sure to run the 1st command before running the  tests:
+Open a separate terminal window #2, execute the following commands
 
 ```
-cargo test --workspace
+make deploy-eigenlayer-contracts
+
+make deploy-helloworld-contracts
 ```
 
-## Existing Holesky Testnet Deployment
+3. Start Operator
 
-| Contract Name               | Holesky Address                                   |
-| -------------               | -------------                                     |
-| Hello World Service Manager | [0x3361953F4a9628672dCBcDb29e91735fb1985390](https://holesky.etherscan.io/address/0x3361953F4a9628672dCBcDb29e91735fb1985390)    |
+```sh
+make start-rust-operator
+```
+4. Spam Tasks
 
-Please see [Current Testnet Deployment](https://github.com/Layr-Labs/eigenlayer-contracts?tab=readme-ov-file#current-testnet-deployment) for additional deployed addresses.
+```sh
+make spam-rust-tasks
+```
 
-You don't need to run a deployment script for holesky testnet, the contracts are already deployed.
+### Testing
 
-1. Use the HOLESKY_ namespace env parameters in the code , instead of normal parameters.
+1. Start Anvil Chain
 
-2. Run `make start-rust-operator`
+In terminal window #1, execute the following commands:
+```sh
+anvil
```

Comment:
This is just an opinion,
This works. However, in the type-script version, it seems we can run everything through `npm` 
It would be more consistent to follow a similar pattern, and execute everything using `make`

Reply:
makes sense, I added the npm scripts because I thought it would be familiar to more devs that this repo was geared towards